### DISCUSSION TITLE
Is it possible to use a VR tracker input with a tracker on the side of my head instead of on top?

### DISCUSSION THREAD

**Discussion by sunaurus** [opentrack/opentrack#1778] [Q&A] [answered]
Hey! 

I am using opentrack 2023.3.0 with a Tundra VR tracker, and it works super well when the tracker is attached flat on the top of my headphone band (center of my head). Unfortunately, this setup is not super stable (it makes my headphones top-heavy). I would like to place the tracker on the side of my head instead, for example attached sideways to one of my headphone ear cups, but I can not figure out how to configure opentrack to correctly interpret such positioning.

With the tracker attached to the side of my head, movements on a single axis get reported as movements on several axes in the "Raw tracker data" box in opentrack, and they generally result in the octopus moving diagonally. For example,  pitching my head directly down makes the octopus pitch and yaw at the same time. 

More specifically, with the tracker attached to the side of my head and me **looking straight ahead**, the "Raw tracker data" shows the following:
![image](https://github.com/opentrack/opentrack/assets/5356547/2063fa0a-a307-47f5-aa4b-50266bb8868c)
**Pitching my head directly down** shows the following:
![image](https://github.com/opentrack/opentrack/assets/5356547/1c64d473-445b-4e7b-bb64-aa980e36a6b2)
And **pitching my head directly up** shows the following:
![image](https://github.com/opentrack/opentrack/assets/5356547/f063ef2a-ec9f-44a4-ac71-48d8349ae857)

Is there any way to compensate for this kind of offset tracker position in opentrack configuration, so the movements could be interpreted correctly?

**Comment by sthalik** [comment]
Maybe it could help to use the non-roll-compensated VR 360 center method.

**Reply by sunaurus** [reply]
Unfortunately this centering method also results in diagonal movement. Is it in general expected for a tracker to work even if it's not oriented "correctly" in the center position?

**Comment by sunaurus** [comment, marked as answer]
I had a go at fixing it: https://github.com/opentrack/opentrack/pull/2103

This solves the issue for me, I am now able to use my Tundra tracker without issues, regardless of how I mount it.
